Week-1/Assignment_1.2/src/App.js

```diff
@@ -0,0 +1,170 @@
+import logo from './logo.svg';
+import './App.css';
+import {useState,useEffect} from "react";
+
+function App() {
+  const [musicList, setmusicList] = useState([
+    {
+      name:"One Dance",
+      author:"Crake feat CatKid & Cyla",
+      duration:"2:54",
+      link:"",
```

Comment:
why is this developed using react? it should be only using HTML and CSS. please remove all react code. will review rest after this change